src/libraries/Microsoft.PowerFx.Core/Functions/FunctionScopeInfo.cs

```diff
@@ -243,7 +251,7 @@ internal class FunctionThisGroupScopeInfo : FunctionScopeInfo
         public static DName ThisGroup => new DName("ThisGroup");
 
         public FunctionThisGroupScopeInfo(TexlFunction function)
-            : base(function, appliesToArgument: (argIndex) => argIndex > 0)
+            : base(function, appliesToArgument: (argIndex) => argIndex > 0, checkPredicateUsage: true)

```

Comment:
>checkPredicateUsage: true [](http://example.com/codeflow?start=76&length=25)

`Summarize` function stands alone in terms of how predicate is used. I'm not sure if we should check it at all. Looking forward to hear from other folks.